.github/workflows/docs.yml

```
@@ -0,0 +1,52 @@
+name: Docs
+
+on:
+  # Update on pushes to main
+  push:
+    branches: ['main']
```

```suggestion
    branches: [main]
```

Would you mind removing the single quotes here? YAML does not require them for strings, and I'd like to be consistent throughout the entire file.